.github/workflows/tests.yml

```diff
@@ -200,6 +200,8 @@ jobs:
             --env-file <(env | egrep 'HOMEBREW|GITHUB') \
             --workdir /tmp/bottles \
             --pull always \
+            --volume /dev/snd:/dev/snd \
+            --privileged \
```

Comment:
I'm not sure we want this?

Reply:
Not something we want.

Just experimenting to see if there was a sound device available on underlying VM to get past current Linux failure.

May just need to skip test (or expect error message) as ALSA either needs an actual device or some `modprobe` commands to enable driver.

Reply:
I think we'll probably want to skip tests that depend on ALSA for now so we can get everything bottled on Linux.  Hopefully we will eventually figure out a way to get proper sound (and graphics) devices enabled in CI so these tests will work, but until we have a solution I think we have to skip them.